cmd/ore/aws/upload.go

```diff
@@ -252,6 +270,15 @@ func runUpload(cmd *cobra.Command, args []string) error {
 				os.Exit(1)
 			}
 		}
+
+		if len(uploadTags) > 0 {
+			err = API.CreateTags([]string{pvID, sourceSnapshot}, tagMap)
```

Comment:
You don't need to retag `sourceSnapshot` here.

Reply:
Thanks!  Fixed.